exchanges/mock/recording_test.go

```diff
@@ -3,39 +3,38 @@ package mock
 import (
 	"net/http"
 	"net/url"
-	"strings"
+	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	"github.com/thrasher-corp/gocryptotrader/encoding/json"
 )
 
 func TestGetFilteredHeader(t *testing.T) {
+	items, err := getExcludedItems()
+	require.NoError(t, err, "getExcludedItems must not error")
+	assert.NotNil(t, items)
```

Comment:
🚧 Sorry, still not addressed.
```suggestion
	assert.NotNil(t, items, "getExcludedItems should not return nil")
```